src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class NumberSupplier {
+}
```

Comment:
Writting this class not in scope of your task, please read requirements more cerafully